recipes/dpdata/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "dpdata" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "53164fd67c5ba2bf2456969eb80adc85c2ca7768957c58716b4b00934de233db" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/py3/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}-py3-none-any.whl
```

Comment:
We prefer to build from the sdist and not the wheel.

Reply:
If so, I need to add a patch because the source code do not contain the version number.

Reply:
Thanks, I gave upstream a hint on how to fix this easily: https://github.com/deepmodeling/dpdata/issues/18